### DISCUSSION TITLE
Is it possible to share configuration between multiple projects in monorepo?

### DISCUSSION THREAD

**Discussion by rxliuli** [egoist/tsup#741] [Ideas]
I have a monorepo with 48 modules and I want to reuse the configuration in multiple submodules, what is the currently recommended way? Is it possible for me to use the config as a lib, like prettier/eslint?

**Comment by Stefano31** [comment]
Here there is a repository (https://github.com/Stefano31/ts-mono) with an example how to share configuration between submodules . In short way, you can create a submodule for example "common-config" and than you can use in others submodules

**Comment by theaungmyatmoe** [comment]
Try `turborepo` which can share config seamlessly
